parsely/build.gradle

```diff
@@ -4,12 +4,23 @@ plugins {
     id 'org.jetbrains.kotlinx.kover'
 }
 
+ext {
+    assertJVersion = '3.24.2'
+    mockWebServerVersion = '4.12.0'
+}
+
 android {
     compileSdkVersion 33
 
     defaultConfig {
         minSdkVersion 21
         targetSdkVersion 33
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunnerArguments clearPackageData: 'true'
+    }
+    testOptions {
+        execution 'ANDROIDX_TEST_ORCHESTRATOR'
```

Comment:
Praise (❤️): TIL about that and the fact that this makes sure that this local storage file won't be shared between test (56c0fe864b332feaf7eae49894b06d8c232b3f95), and if removed, this will make the test to fail as the local storage file name will now exist, interesante! 🧪